src/components/RelatedDocumentsList/RelatedDocumentsListPresentation.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Link } from 'react-router'
+
+// FPCC
+import getIcon from 'common/utils/getIcon'
+
+function RelatedDocumentsListPresentation({
+  documents,
+  labelStyling = 'text-left font-medium text-lg uppercase text-charcoal-900',
+}) {
+  return (
+    documents?.length > 0 && (
+      <table className="w-full">
```

Comment:
I defer to @gmcauliffe on this but using a table element just for layout is not semantic and typically not preferred. If we are going to stick with a table, let's at least add the necessary accessibility features like scope for the th cells and a caption.